packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -63,10 +84,10 @@ export const getComponentCss = (
         // to override inline styles set by splide library
         ...(wrapContent &&
           addImportantToEachRule({
-            // TODO: 0 calc(${gridSafeZone} + ${gridGap})
-            padding: `0 calc(${gridSafeZone} + 7%) 0 ${gridSafeZone}`,
+            // TODO: 0 calc(${gridSafeZone} + ${gridGap}) - will be done after Grid Refactoring
+            padding: `0 calc(${gridSafeZone} + ${spacingFluidLarge}) 0 ${gridSafeZone}`,
             [mediaQueryXl]: {
-              padding: `0 calc(${gridSafeZone} + 7%) 0 ${gridSafeZone}`,
+              padding: `0 calc(${gridSafeZone} + ${spacingFluidLarge}) 0 ${gridSafeZone}`,
```

Comment:
Identical to previous breakpoint or even base styles and should therefore be removed.

Reply:
Fixed it